### DISCUSSION TITLE
A JavaScript error occurred in the main process

### DISCUSSION THREAD

**Discussion by ivanjay233** [upscayl/upscayl#602] [Q&A]
Hi, can anyone please help me?
this error occured to me

Uncaught Exception:
URIError: URI malformed
at decodeURIComponent (<anonymous>)
at Function.<anonymous> (/Applications/Upscayl.app/Contents/Resources/app.asar/node_modules/electron-next/index.js:63:12)

what should i do?

**Comment by ivanjay233** [comment]
LOG :
🎯 GET_MODELS_LIST:  
⚙️ Getting saveImageAs from localStorage:  jpg
⚙️ Getting model from localStorage:  {"label":"General Photo (Ultrasharp)","value":"ultrasharp"}
⚙️ Setting gpuId to empty string
🔀 Setting model to 
🔀 Setting model to ultrasharp
🐞 BACKEND REPORTED:  💾 Updating Save Output Folder:  false
🐞 BACKEND REPORTED:  📐 Updating Compression:  NaN
🐞 BACKEND REPORTED:  📝 Updating Overwrite:  false
⚙️ Getting saveImageAs from localStorage:  jpg
⚙️ Getting model from localStorage:  {"label":"General Photo (Ultrasharp)","value":"ultrasharp"}
⚙️ Setting gpuId to empty string
⚙️ Getting rememberOutputFolder from localStorage:  false
⚙️ Getting saveImageAs from localStorage:  jpg
⚙️ Getting model from localStorage:  {"label":"General Photo (Ultrasharp)","value":"ultrasharp"}
⚙️ Setting gpuId to empty string
🔀 Setting model to 
🔀 Setting model to ultrasharp
🔄 Resetting image paths
🐞 BACKEND REPORTED:  🖼️ Updating Image Path:  /Users/ivan/Desktop/untitled folder/70%/1.png
🖼 Selected Image Path:  /Users/ivan/Desktop/untitled folder/70%/1.png
📁 Selected Image Directory:  /Users/ivan/Desktop/untitled folder/70%
🖼 imagePath:  /Users/ivan/Desktop/untitled folder/70%/1.png
🔤 Extension:  png
🐞 BACKEND REPORTED:  📄 Selected File Path:  /Users/ivan/Desktop/untitled folder/70%/1.png
⚙️ Getting saveImageAs from localStorage:  jpg
⚙️ Getting model from localStorage:  {"label":"General Photo (Ultrasharp)","value":"ultrasharp"}
⚙️ Setting gpuId to empty string
⚙️ Getting rememberOutputFolder from localStorage:  false

**Comment by NayamAmarshe** [comment]
Maybe change the name of the 70% folder?

**Reply by ivanjay233** [reply]
sorry, still can't 
<img width="1256" alt="Screen Shot 2023-12-17 at 20 37 17" src="https://github.com/upscayl/upscayl/assets/112231746/edd39ac0-692f-42e4-ab29-4895cb5a1a0d">
any idea again?

**Reply by NayamAmarshe** [reply]
We'll need the full logs. Please file a new issue instead of discussion post.

**Comment by aaronliu0130** [comment]
Please file an issue and file the full logs.
